portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -57,16 +78,20 @@ public void doPost(HttpServletRequest request, HttpServletResponse response) thr
     String email = getParameter(request, "email", "");
     String subject = getParameter(request, "subject", "");
     String comments = getParameter(request, "comments", "");
-    
-    messages = new ArrayList<String>();
-    messages.add(name);
-    messages.add(email);
-    messages.add(subject);
-    messages.add(comments);
+    String timestamp_UTC = new Timestamp(System.currentTimeMillis()).toString();
+
+    // Store comments.
+    Entity commentEntity = new Entity("Comment");
+    commentEntity.setProperty("timestamp_UTC", timestamp_UTC);
```

Comment:
Could you please define all the string constants as class variable.
see: https://google.github.io/styleguide/javaguide.html#s5.2.4-constant-names